javascript/.eslintrc.json

```
@@ -0,0 +1,12 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6": true,
+        "node": true
```

```suggestion
        "commonjs": true,
        "es6": true
```
We don't need node-specific or browser-specific features at present.